api/envoy/api/v2/cluster/circuit_breaker.proto

```diff
@@ -46,6 +46,10 @@ message CircuitBreakers {
     // The maximum number of parallel retries that Envoy will allow to the
     // upstream cluster. If not specified, the default is 3.
     google.protobuf.UInt32Value max_retries = 5;
+
+    // The maximum number of connection pools that Envoy will concurrently support at once.
```

Comment:
Maybe be explicit that this is per cluster like the comments for the other circuit breakers are?